openpype/hosts/max/plugins/publish/validate_resolution_setting.py

```diff
@@ -12,7 +12,7 @@ class ValidateResolutionSetting(pyblish.api.InstancePlugin,
     """Validate the resolution setting aligned with DB"""
 
     order = pyblish.api.ValidatorOrder - 0.01
-    families = ["maxrender"]
+    families = ["maxrender", "review"]
```

Comment:
Are we sure this validation is still accurate- the extract review doesn't use `rt.renderwidth` right? It uses the defined "resolution" which it applies in a context manager to render at that resolution instead. This validation seems off (for reviews at least)

Reply:
I actually forgot to remove this. It was supposed that user would use the render resolution before extraction